### ISSUE TITLE
allow ingredients withou amount

### ISSUE THREAD

**Issue by mkaut** [nextcloud/cookbook#2233]
In many recipes, the amount of seasoning (salt, pepper, etc) is often unspecified, with the understanding that this means "according to your taste".
If I do this in the Cookbook, the ingredient gets marked with an exclamation mark and a warning that it cannot be recalculated - which i only annoying - it is kind of obvious that if there is no number, it does not get recalculated, there is no need for a warning.

Could we get an option for removing the warning?

**Comment by grantmulholland** [comment]
Yep, just ran into this myself. I'd rather not have to explain that the error is fine to ignore.

**Comment by christianlupus** [comment]
Yeah, the point is that the recalculation might fail in other situations as well. This exclamation mark is a warning that the line was not parsable. However, one could add a config option to disable it.

**Comment by jjhuff** [comment]
@christianlupus I'm not sure how hard it'd be but, extending the data model (and parser) to support ranges and optional/unparsable quantity would be awesome.

**Comment by lmarvaud** [comment]
Hello,

A solution may be to use `0` (zero) as the amount and to hide this amount in the list. This solution is use by the app cookbookmanager.com (cf. [What do I do if my ingredient doesn't have a quantity?](https://help.cookbookmanager.com/hc/en-gb/articles/360002691615-What-do-I-do-if-my-ingredient-doesn-t-have-a-quantity) in there help center).

This allow the user to manually forcefully indicate to the system that there is no specific amount for this ingredient. A technical solution would be to add a special character (`#`, `$`, or `0` (zero)...) configurable.
